src/Logger/Adapter/HoneyBadger.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace Utopia\Logger\Adapter;
+
+use Exception;
+use Utopia\Logger\Adapter;
+use Utopia\Logger\Log;
+use Utopia\Logger\Logger;
+
+// Reference Material
+// https://docs.honeybadger.io/api/reporting-exceptions
+
+class HoneyBadger extends Adapter
+{
+    protected string $apiKey;
+
+    /**
+     * Return unique adapter name
+     *
+     * @return string
+     */
+    public static function getName(): string
+    {
+        return 'honeyBadger';
+    }
+
+    /**
+     * Push log to external provider
+     *
+     * @param  Log  $log
+     * @return int
+     */
+    public function push(Log $log): int
+    {
+        $params = [];
+
+        foreach ($log->getExtra() as $paramKey => $paramValue) {
+            $params[$paramKey] = var_export($paramValue, true);
+        }
+
+        $breadcrumbsObject = $log->getBreadcrumbs();
+        $breadcrumbsArray = [];
+
+        foreach ($breadcrumbsObject as $breadcrumb) {
+            \array_push($breadcrumbsArray, [
+                'category' => $breadcrumb->getCategory(),
+                'timestamp' => \intval($breadcrumb->getTimestamp()),
```

Comment:
I tested this recently, but got an error in Honeybadger because they expect this to be a string like `2020-01-27T12:37:31.616-08:00`. Would you please update this to be a ISO-8601 formatted timestamp in UTC?